### ISSUE TITLE
Finding four corners?

### ISSUE THREAD

**Issue by khushaljethava** [lab231/ST-RoomNet#3]
Hello, Great work,
Is there a way to find the 4 corners of the floor?
![result (1)](https://github.com/lab231/ST-RoomNet/assets/12109822/8dffb3f7-8285-407e-bb46-5017265c4112)
This is the image and we can see only one corner is visible but I want to find the 4 corners, I have tried to find the intersection point but it's not perfect.

![OTlhv](https://github.com/lab231/ST-RoomNet/assets/12109822/fa20acc8-a32e-4d14-812c-94694f18ee3b)
This is the reference image, I want to find 4 corners like this.

I hope you can help me with this. Thanks in advance.

**Comment by ebrahimkhalilabbasi** [comment]
Hello @khushaljethava,

Have you managed to find the corners?

**Comment by khushaljethava** [comment]
Hello @ebrahimkhalilabbasi,

I haven't had much success, but you can determine the estimated corners by identifying the intersection points.

**Comment by ebrahimkhalilabbasi** [comment]
Does the code give the geometry data as output?

**Comment by khushaljethava** [comment]
No, it does not, and I believe it's challenging to extract geometry data from just a single image. You can try training a model to do that.

**Comment by ebrahimkhalilabbasi** [comment]
@khushaljethava Thanks for your answer. The output does not have the corresponding length and width of the walls, ceiling, and floors. How is it possible to find the intersections so?
